app/components/primer/beta/auto_complete.rb

```diff
@@ -22,13 +32,26 @@ class AutoComplete < Primer::Component
         system_arguments[:tag] = :ul
         system_arguments[:id] = @list_id
         system_arguments[:classes] = class_names(
-          "autocomplete-results",
+          "ActionList",
           system_arguments[:classes]
         )
 
         Primer::BaseComponent.new(**system_arguments)
       }
 
+      #
+      # Leading visual.
+      #
+      # - `leading_visual_icon` for a <%= link_to_component(Primer::OcticonComponent) %>.
+      #
+      # @param system_arguments [Hash] Same arguments as <%= link_to_component(Primer::OcticonComponent) %>.
+      renders_one :leading_visual, types: {
```

Comment:
Is an octicon the only possible type of leading visual?

Reply:
Yup!